.github/workflows/aws-template-terraform.yml

```
@@ -59,16 +59,23 @@ jobs:
   scan:
     name: Scan TF Code
     runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
       - name: tfsec
-        uses: aquasecurity/tfsec-action@v1.0.3
+        uses: aquasecurity/tfsec-sarif-action@21ded20e8ca120cd9d3d6ab04ef746477542a608
```

```suggestion
        uses: aquasecurity/tfsec-sarif-action@v0.1.4
```

I'd recommend using the tagged version rather than the commit hash.  This makes it easier for automated tools such as Dependabot to manage updates if we were to introduce this sort of configuration; https://github.com/bcgov/von-network/blob/main/.github/dependabot.yml#L5-L14

---

This make sense. I applied the recommendation. Thank you.